sopel/modules/url.py

```diff
@@ -18,8 +18,8 @@
 import dns.resolver
 import requests
 
-from sopel import module, tools
-from sopel.config.types import ListAttribute, StaticSection, ValidatedAttribute
```

Comment:
🥖crumb number I-forgot-how-many-of-each-there-are about config types import consistency.